auth0/v3/management/users.py

```diff
@@ -66,6 +66,27 @@ def list(self, page=0, per_page=25, sort=None, connection=None, q=None,
         }
         return self.client.get(self._url(), params=params)
 
+    def list_by_email(self, email, fields=[], include_fields=True):
+        """Get a List of Users by E-Mail.
+
+        Args:
+            email (str): The email of the users to retrieve.
+
+            fields (list of str, optional): A list of fields to include or
+                exclude from the result (depending on include_fields). Empty to
+                retrieve all fields.
+
+            include_fields (bool, optional): True if the fields specified are
+                to be include in the result, False otherwise.
+        """
+        params = {
+            'email': email,
```

Comment:
Per the comment [here](https://auth0.com/docs/api/management/v2#!/Users_By_Email/get_users_by_email) this should be `email.lower()`

>When using this endpoint, make sure that you are searching for users via email addresses in lowercase only.